lib/statsd.js

```diff
@@ -42,18 +43,28 @@ var Client = function (host, port, prefix, suffix, globalize, cacheDns, mock,
       maxBufferSize : maxBufferSize,
       bufferFlushInterval: bufferFlushInterval,
       telegraf    : telegraf,
-      sampleRate    : sampleRate
+      sampleRate  : sampleRate,
+      protocol    : protocol
     };
   }
 
+  function createSocket(args) {
```

Comment:
I'd use `options` instead of `args`, but both are saying nothing about the contents, so it's just a convention nitpick.